docs/book/src/tasks/kubeadm-control-plane.md

```diff
@@ -16,7 +16,24 @@ See the section on [upgrading clusters][upgrades].
 
 See the section on [Adopting existing machines into KubeadmControlPlane management][adoption]
 
+### Running workloads on control plane machines
+
+The user may want to run non-control plane workloads on control plane machines.
```

Comment:
We should probably mention that we don't suggest to run workloads on control planes, highly encourage to avoid it unless absolutely necessary

Reply:
Done, PTAL